scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -587,6 +589,42 @@ class ScioContext private[scio] (val options: PipelineOptions,
       }
     }
 
+  /**
+    * Get an SCollection for a CloudSql query
+    * @group input
+    */
+  def cloudSqlSelect[T: ClassTag](readOptions: JdbcReadOptions[T]): SCollection[T] = {
```

Comment:
If it doesn't have an impact apart from method name, let's keep just one, to keep things simple?